core/shared_queue.js

```diff
@@ -189,6 +207,10 @@ SharedQueue Binary Layout
       push,
       reset,
       shift
+    },
+    ops: {
+      init: initOps,
+      get: getOpId
```

Comment:
`Deno.core.ops.get` is becoming distastefully long.

How about `Deno.core.ops()` returns the map. Remove the "init".